contribute/style-guides/documentation-style-guide.md

```diff
@@ -300,6 +298,19 @@ One word, not two.
 
 One word, not two. Also, not hyphenated.
 
+#### MS SQL Server
```

Comment:
Looks like SQL Server is the correct trademark, per https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWArzw.